src/main/java/core/basesyntax/PrintArray.java

```diff
@@ -0,0 +1,19 @@
+package core.basesyntax;
+
+import core.basesyntax.model.Figure;
+import core.basesyntax.supplier.FigureSupplier;
+
+public class PrintArray {
+    private static final FigureSupplier FIGURE_SUPPLIER = new FigureSupplier();
```

Comment:
```suggestion
    private static final FigureSupplier figureSupplier = new FigureSupplier();
```
FigureSupplier - это объект, который может быть изменен в ходе работы программы. То есть это не константа.
![image](https://user-images.githubusercontent.com/20326858/214825414-61fd6a38-d3a8-4943-85c9-5bb07e1e58a2.png)